### PR TITLE
Adding new Slot: commentAuthorTags

### PR DESCRIPTION
**Pull request by okbel**
## What does this PR do?
- Story:   https://www.pivotaltracker.com/story/show/148441167

- Adds a new Slot `commentAuthorTags` (named by @IAmSamHankins )
- New Recipe: Subscriber Badge https://github.com/coralproject/talk-recipes/pull/3

### CODE REVIEW

File: client/coral-embed-stream/src/components/Comment.js [hunk 1]
```diff
@@ -447,6 +447,14 @@ export default class Comment extends React.Component {
 
               {isStaff(comment.tags) ? <TagLabel>Staff</TagLabel> : null}
 
+              <Slot
+                className={cn('talk-stream-comment-author-tags')}
+                fill="commentAuthorTags"
+                queryData={queryData}
+                {...slotProps}
+                inline
```
Comment (alinhle) [inline client/coral-embed-stream/src/components/Comment.js:455]: I'd love to deprecated the `inline` property and just use `className` to pass styling into the Slot instead. wdyt?

Comment (okbel) [reply client/coral-embed-stream/src/components/Comment.js:455]: TBH I love `inline` it makes it so much easier to develop inline slots. Any specific reason for it?

**Comment by okbel** [comment]
Updated with `excludeIf` !

File: client/coral-embed-stream/src/components/Comment.js (hunk 1 above)
Comment (okbel) [reply client/coral-embed-stream/src/components/Comment.js:455]: I'd say we should merge this so we have the recipe up and running. And we can revisit this later and all together in another PR.

Comment (alinhle) [reply client/coral-embed-stream/src/components/Comment.js:455]: Yeah it's not a big issue, just wanting to hear your thoughts, and trying to have a clear concept on the `inline` property, which I don't have.

**Review by wyattjoh — APPROVED** [review]

**Pull request merged**